monkey/monkey_island/cc/resources/island_logs.py

```diff
@@ -1,19 +1,22 @@
 import logging
+from pathlib import Path
 
 from monkey_island.cc.resources.AbstractResource import AbstractResource
 from monkey_island.cc.resources.request_authentication import jwt_required
-from monkey_island.cc.services.island_logs import IslandLogService
+from monkey_island.cc.server_utils.island_logger import get_log_file_contents
 
 logger = logging.getLogger(__name__)
 
 
 class IslandLog(AbstractResource):
-    # API Spec: Why the inconsistency in endpoints of IslandLog and Log?
-    urls = ["/api/log/island/download"]
+    urls = ["/api/island/log"]
+
+    def __init__(self, island_log_file_path: Path):
+        self._island_log_file_path = island_log_file_path
 
     @jwt_required
     def get(self):
         try:
-            return IslandLogService.get_log_file()
+            return {"log_file": get_log_file_contents(self._island_log_file_path)}
```

Comment:
```suggestion
            return {"log_contents": get_log_file_contents(self._island_log_file_path)}
```

Reply:
Just return a string. The resource tells you what it is. Putting {"log_contents:" ...} is redundant.

Reply:
Unless it has to be JSON?

Reply:
It looks like a single string may be valid JSON.

![image](https://user-images.githubusercontent.com/19957806/182159717-1f785006-26d2-4ddd-aad7-6b0c1ef500a8.png)

Reply:
![image](https://user-images.githubusercontent.com/19957806/182159915-928fa001-72a7-471b-a00a-f00dc158b9d0.png)

Reply:
Tested and it works